src/pages/dex/[poolpairId]/_components/PoolPairDexStabilizationFee.tsx

```diff
@@ -0,0 +1,23 @@
+import ReactNumberFormat from 'react-number-format'
+import BigNumber from 'bignumber.js'
+import { InfoHoverPopover } from '@components/commons/popover/InfoHoverPopover'
+
+export function PoolPairDexStabilizationFee (): JSX.Element {
+  return (
+    <div className='flex flex-wrap flex-row mt-8 p-4 lg:p-6 rounded-lg border border-gray-200 justify-between items-center dark:border-gray-700 dark:bg-gray-800' data-testid='PoolPairDexStabilizationFee'>
+      <div className='flex items-center mr-4 lg:mr-8'>
+        <span className='mr-2 sm:whitespace-nowrap text-gray-900 dark:text-gray-100'>DEX stabilization fee</span>
+        <InfoHoverPopover description='There is currently a high DEX Stabilization fee imposed on DUSD-DFI swaps due to DFIP 2206-D.' />
+      </div>
+      <ReactNumberFormat
+        displayType='text'
+        className='font-medium md:text-xl dark:text-gray-100'
+        thousandSeparator
+        decimalScale={2}
+        fixedDecimalScale
+        suffix='%'
+        value={new BigNumber(29.74).toFixed(2, BigNumber.ROUND_HALF_UP)}
```

Comment:
https://github.com/JellyfishSDK/jellyfish/blob/905a3e741587c69b3f07cb89bd838aa6c1bb698b/packages/whale-api-client/src/api/poolpairs.ts#L119-L123

You can use this for DEX fee.

Reply:
> https://ocean.defichain.com/v0/mainnet/poolpairs/17

```json
{"data":{"id":"17","symbol":"DUSD-DFI","displaySymbol":"DUSD-DFI","name":"Decentralized USD-Default Defi token","status":true,"tokenA":{"symbol":"DUSD","displaySymbol":"DUSD","id":"15","reserve":"32502083.03787209","blockCommission":"0","fee":{"pct":"0.2974","inPct":"0.2974"}},"tokenB":{"symbol":"DFI","displaySymbol":"DFI","id":"0","reserve":"34938213.86058497","blockCommission":"0"},"priceRatio":{"ab":"0.93027317","ba":"1.07495306"},"commission":"0.002","totalLiquidity":{"token":"32761986.99611493","usd":"65004166.07574418"},"tradeEnabled":true,"ownerAddress":"8UAhRuUFCyFUHEPD7qvtj8Zy2HxF5HH5nb","rewardPct":"0","creation":{"tx":"8334fa67b87332d658fc77b15ba09b8721c0a64f9a470441a449a784c7e97a68","height":1367574},"apr":{"reward":0.42518887351858514,"commission":0.0655230369117309,"total":0.4907119104303161},"volume":{"h24":5834616.949585304,"d30":150874279.69346902}}}
```